tests/ExcelTest.php

```diff
@@ -79,11 +85,17 @@ public function can_store_an_export_object_on_default_disk()
     public function can_store_an_export_object_on_another_disk()
     {
         $export = new EmptyExport;
+        $name   = 'filename.xlsx';
+        $path   = __DIR__ . '/Data/Disks/Test/' . $name;
```

Comment:
Can we perhaps wrap this into a helper function so we don't have to repeat the same code